Tests/ApolloTests/BatchedLoadTests.swift

```diff
@@ -53,16 +53,21 @@ private final class MockBatchedNormalizedCache: NormalizedCache {
 class BatchedLoadTests: XCTestCase {  
   func testListsAreLoadedInASingleBatch() {
     var records = RecordSet()
-    let drones = (1...100).map { number in
-      Record(key: "Drone_\(number)", ["__typename": "Droid", "name": "Droid #\(number)"])
-    }
+    let drones = (1...100)
+      .map { Record(key: "Drone_\($0)", ["__typename": "Droid", "name": "Droid #\($0)"]) }
```

Comment:
I think it would help to leave the named parameter on this for clarity, especially since you're also referring to a `$0` parameter in the 2nd map here.

Reply:
Cleaned this up